core/modules/user/user.module

```diff
@@ -2482,7 +2482,7 @@ function user_role_load_by_name($role_name) {
 function user_role_save($role) {
   // Trim spaces from names.
   $role->name = trim($role->name);
-  $role->label = trim($role->label);
+  $role->label = check_plain(trim($role->label));
```

Comment:
This change looks incorrect. If you save a role that includes an HTML entity like `&` then when you re-edit the role it will show the encoded version instead, saving again would then double-encode, triple-encode, etc with each save. I don't think this change is necessary to accomplish the objective, it should work fine without it.